playground/CosmosEndToEnd/CosmosEndToEnd.AppHost/Program.cs

```diff
@@ -4,8 +4,9 @@
 var builder = DistributedApplication.CreateBuilder(args);
 
 var db = builder.AddAzureCosmosDB("cosmos")
-                .AddDatabase("db")
-                .RunAsEmulator();
+                .WithDatabase("db", database => database.Containers.Add(new("entries", "/Id")))
+                .RunAsEmulator()
+                ;
```

Comment:
```suggestion
                .RunAsEmulator();
```

This looks like a mistake